package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "root",
+  "private": true,
+  "devDependencies": {
+    "husky": "^4.2.5",
+    "lerna": "^3.22.1"
+  },
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "start": "lerna run start"
```

Comment:
shouldn't we be using --since option here?

Reply:
Nope we will be using --since option inside husky like below:
"husky": {
    "hooks": {
      "pre-commit": "lerna run pre-commit --since master",
      "pre-push": "lerna run pre-push --since master"
    }
  }